content/en/docs/instrumentation/python/automatic.md

```diff
@@ -172,7 +172,7 @@ Stop the execution of `server_instrumented.py` by pressing <kbd>Control+C</kbd>
 and run the following command instead:
 
 ```console
-$ opentelemetry-instrument --traces_exporter console python server_uninstrumented.py
+$ opentelemetry-instrument --traces_exporter console --metrics_exporter console python server_uninstrumented.py
```

Comment:
hmmm, does this also work with `--metrics_exporter none`? Would prefer to not list an exporter for metrics unless there's metrics to export. And I recall some folks saying this shouldn't error our but I guess it still is :/

Reply:
It does work with `none`.  I'll update the PR.

Reply:
Done